pkg/section/local_module.go

```diff
@@ -38,40 +36,21 @@ func (m *LocalModule) Type() string {
 // Configure configures the module section by finding the module
 // for the current path
 func (m *LocalModule) Configure() error {
-	modPaths, err := findLocalModules()
+	modPath, err := findLocalModule()
 	if err != nil {
 		return err
```

Comment:
It might be helpful to explain in the error why we were doing whatever that failed, e.g. if a user just copies some config from a single module repo to one using workspaces this would start failing and it might not be obvious why.

```suggestion
		return fmt.Errorf("failed finding local modules for `localModule` configuration: %w", err)
```